sys-libs/libatomic_ops/libatomic_ops-7.4.2.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided \
+atomic memory update operations on a number architectures \
+as said on the github project home."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops/"
+COPYRIGHT="2003-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.ivmaisoft.com/_bin/atomic_ops/libatomic_ops-7.4.2.tar.gz"
+CHECKSUM_SHA256="04fa615f62992547bcbda562260e28b504bc4c06e2f985f267f3ade30304b5dd"
+
+ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+       libatomic_ops$secondaryArchSuffix = 1.0.3 compat >= 1
```

Comment:
You were supposed to change `$portVersion` only for provided libraries. This entry should still have `$portVersion`.